### PR TITLE
Update upgrading-from-260-to-400.md

### PR DESCRIPTION
**Pull request by samgnaniah**
Fixes https://github.com/wso2-enterprise/wso2-apim-internal/issues/359

### CODE REVIEW

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-400.md [hunk 1]
```diff
@@ -3651,6 +3651,16 @@ Follow the instructions below to move all the existing API Manager configuration
 
     6.  Stop the WSO2 API-M server and remove the `tenantloader-1.0.jar` from the `<API-M_4.0.0_HOME>/repository/components/dropins` directory.
 
+8. If you already have used secondary user stores in API-M 2.6.0, you have to copy the user store file created inside the old API-M version to the new API-M version.
+
+    For secondary user stores created for the super tenant,
+    
+    You have to copy the `<API-M_2.6.0_HOME>/repository/deployment/server/userstores/` directory inside `<API-M_4.1.0_HOME>/repository/deployment/server/` directory.
```
Comment (msm1992) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-400.md:3658]: API-M version has to be corrected as 4.1.0

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-400.md [hunk 2]
```diff
@@ -3651,6 +3651,16 @@ Follow the instructions below to move all the existing API Manager configuration
...
+    
+    For secondary user stores created for tenants,
+    
+    You have to copy the userstores to respective tenant directories after executing [step 4](#step-4-migrate-the-api-manager-components) as the tenant directories (`<API-M_2.6.0_HOME>/repository/tenants/`) at API-M 4.1.0 are populated after migrating the api-m components.
```
Comment (msm1992) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-400.md:3662]: API-M version has to be corrected as 4.0.0

Comment (samgnaniah) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-400.md]: Fixed with [408dd54](https://github.com/wso2/docs-apim/pull/6171/commits/408dd5414e34d0a3f6fcc07e07bf294039a5b7f4)

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-400.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-400.md]: Fixed with [408dd54](https://github.com/wso2/docs-apim/pull/6171/commits/408dd5414e34d0a3f6fcc07e07bf294039a5b7f4)

**Review by msm1992 — APPROVED** [review]

**Pull request merged**